package.json

```diff
@@ -17,6 +17,7 @@
     "react-modal": "3.11.2",
     "react-router-dom": "5.1.2",
     "react-scripts": "3.2.0",
+    "react-select": "^3.1.0",
```

Comment:
Please remove `^`. This ensures that we lock in the library version and don't accidentally cause any issues down the line.